app/AppComponents.scala

```diff
@@ -20,8 +21,13 @@ class AppComponents(context: Context, identity: AppIdentity) extends BuiltInComp
 
   val config = new AppConfig(configuration, identity)
 
-  val permissionsClient = new Permissions(config.stage)
-  logger.info(s"Permissions object initialised with config: ${permissionsClient.config}")
+  val permissionsConfig = PermissionsConfig(
+    stage = config.stage,
+    region = config.region,
+    awsCredentials = credentialsV1
+  )
+  val permissionsClient = PermissionsProvider(permissionsConfig)
+  logger.info(s"Permissions object initialised with config: $permissionsConfig")
```

Comment:
is there a risk of this leaking secrets (now or in future if the config contains something sensitive in future)?

Reply:
Yes. I was aiming for parity, but I can remove this